### PR TITLE
completed day 1 exercises of advant of code 2021 in rust

### PR DESCRIPTION
**Pull request by ghost**
Mark X to agree:
- I have made sure my code is inline with the [code guidelines](https://docs.avdanos.com/docs/category/code-guidelines) [X]
- I have updated all readme files that need to be updated [X]

### CODE REVIEW

**Comment by ghost** [comment]
Well this didn't last long lmao

File: README.md [hunk 1]
```diff
@@ -21,3 +21,9 @@ Made by [TechStudent11](https://github.com/TechStudent11)
 A simple 90s-stylish website written in Rust that prints "Hello world" using the Yew Library
 
 Made by [akane](https://github.com/akane6704)
+
+**Advent of code**
+    **day 1**
+        completed day 1 exercises in rust
+
+        Made by [Padmanabh](https://github.com/Padmanabh82)
```
Comment (Froxcey) [inline README.md:29]: The README addition looks kinda odd and out of place

**Review by Froxcey — CHANGES_REQUESTED** [review]
I left a comment, please review and make changes

**Comment by ghost** [comment]
Btw aren't you discontinuing the learning of rust? @Padmanabh82

**Comment by Froxcey** [comment]
It was made two days ago...

File: README.md (hunk 1 above)
Comment (MrSerge01) [reply README.md:29]: I agree.

**Comment by ghost** [comment]
Well since that happened, closing

**Comment by ghost** [comment]
goos i hate you

**Comment by MrSerge01** [comment]
thank you for hating me, very appreciated

**Comment by ghost** [comment]
lmfao

**Comment by ghost** [comment]
@akane6704, action appreciated

File: README.md (hunk 1 above)
Comment (AZProductions) [reply README.md:29]: Same here

**Comment by AZProductions** [comment]
This pr is unsafe, it has a Readme, Readme's are not safe, write it in rust

**Comment by MrSerge01** [comment]
rewrite markdown on Rust

**Pull request closed without merging**